src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -204,6 +204,12 @@ public static void CopyToCovariantArray()
         [InlineData((4L * 1024L * 1024L * 1024L) + 256)]
         public static void CopyToLargeSizeTest(long bufferSize)
         {
+            // If this test is run in a 32-bit process, the large allocation will fail.
+            if (Unsafe.SizeOf<IntPtr>() != sizeof(long))
```

Comment:
I followed the examples here:
https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Span/Overflow.cs#L25

> Or just use `IntPtr.Size == 8`. 

I could do:
https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Span/Clear.cs#L233

What is the advantage of doing that?